src/library/scala/concurrent/Future.scala

```diff
@@ -101,9 +107,13 @@ trait Future[+T] extends Awaitable[T] {
    *  If the future has already been completed with a value,
    *  this will either be applied immediately or be scheduled asynchronously.
    *
+   *  Note that the returned value of `pf` will be discarded.
+   *
    *  $swallowsExceptions
    *  $multipleCallbacks
    *  $callbackInContext
+   *
+   * @group Callbacks
```

Comment:
@janekdb `andThen` does not modify the result of the returned Future, it merely injects an effect between `this`-Future and `that`-Future (where this is the Future it is invoked upon and that is the Future which is returned by the invokation)